### PR TITLE
Add binding for single-part digest function

### PR DESCRIPTION
**Pull request by ellerh**

### CODE REVIEW

File: cryptoki/src/session/digesting.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Contributors to the Parsec project.
```
Comment (ionut-arm) [inline cryptoki/src/session/digesting.rs:1]:
```suggestion
// Copyright 2023 Contributors to the Parsec project.
```

**Review by ionut-arm — APPROVED** [review]
Hi, thanks for the patch, looks good to me! Only one change requested, and I think you need to provide a sign-off for the commit.

**Comment by adamjohnstonarm** [comment]
I would suggest squashing these commits when merging to keep the copyright date clean.

**Review by wiktor-k — APPROVED** [review]
Indeed. Looks good 👍

**Comment by ionut-arm** [comment]
@hcldan - please see #160. Unfortunately, the many changes that have occurred since 0.4.1 means we have to publish a 0.5.0 version now.

**Pull request merged**